addons/mail/static/tests/tours/discuss_channel_public_tour.js

```diff
@@ -122,7 +122,13 @@ registry.category("web_tour.tours").add("discuss_channel_public_tour.js", {
         {
             content: "Check edited message contains the extra attachment",
             trigger: '.o-mail-Message .o-mail-AttachmentCard:contains("extra.txt")',
-            run() {},
+            async run() {
+                await click(".o-mail-AttachmentCard-unlink", {
+                    parent: [".o-mail-AttachmentCard", { text: "extra.txt" }],
+                });
+                await click(".modal .btn.btn-primary");
+                await contains(".o-mail-AttachmentCard", { text: "extra.txt", count: 0 });
+            },
```

Comment:
Took me a bit of time to understand the "primary" button is to confirm a deletion. Would have been easier with a selector a bit more clearer

Reply:
The text is like "ok", I'm not sure if it's a lot more clear, and there are no specific classes to help that I saw.